CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 * Increased the minimum versions of `MapboxNavigationNative` to v30.0 and `MapboxCommon` to v9.2.0. ([#2793](https://github.com/mapbox/mapbox-navigation-ios/pull/2793))
 * Fixed an issue that caused the App Store to reject some application submissions with error ITMS-90338. ([#2793](https://github.com/mapbox/mapbox-navigation-ios/pull/2793))
+* The CarPlay guidance panel now shows lane guidance. ([#1885](https://github.com/mapbox/mapbox-navigation-ios/pull/1885))
+* Fixed an issue where lane guidance icons would indicate the wrong arrow for certain maneuvers. ([#2796](https://github.com/mapbox/mapbox-navigation-ios/pull/2796)) 
```

Comment:
v1.2.1 has already been released. If you rebase atop main, there’s a new v1.3.0 section that these changes can go in. (Technically I should’ve called it “main” but v1.3.0 is probably where this change would end up.)

Reply:
Rebased the PR and moved the changelog entry.

Reply:
Also I reverted the change of the method parameter names. No real need to change it if it's from the PaintCode generated code.